### PR TITLE
Bump Crystal version

### PR DESCRIPTION
**Pull request by straight-shoota**

### CODE REVIEW

**Review by carlhoerberg — APPROVED** [review]

**Comment by straight-shoota** [comment]
@schovi would be great to get this merged.

**Comment by straight-shoota** [comment]
@schovi is not responding unfortunately. With Crystal 1.0 release around the corner, it's important to get this merged.
If someone else could open a PR with this change (maybe @jwoertink, @f4tq, @Sija, @carlhoerberg?) I can merge it. I have merge access but merging requires an approval and I can't approve my own PR.

**Comment by jwoertink** [comment]
Sweet. Thanks for merging that @schovi!

**Pull request merged**
